src/modules/chat_custom_timeouts/index.js

```diff
@@ -129,6 +137,13 @@ function handleClick(e) {
 class ChatCustomTimeoutsModule {
     constructor() {
         watcher.on('load.chat', () => this.loadClickHandler());
+
+        settings.add({
```

Comment:
my last question is should this be a setting. the settings menu is getting long, i'd prefer less options

Reply:
It was my thought that some people wouldn't want an intrusive popup every time and just wanted it as it was, but anyone who does want to add reasons can turn it on.